components/DailyStatsDashboard.tsx

```diff
@@ -0,0 +1,80 @@
+import React from 'react'
+import { Button } from 'react-bootstrap'
+import { Sync } from '@styled-icons/fa-solid/Sync'
+import { useAuth0 } from '@auth0/auth0-react'
+import useSWR, { mutate } from 'swr'
+
+import DailyStatsChart from './DailyStatsChart'
+import FetchMessage from './FetchMessage'
+
+const DAILY_STATS_URL = `${process.env.API_BASE_URL}/api/secure/dailystats?fromDate=2025-01-01`
```

Comment:
```suggestion
const DAILY_STATS_URL = `${process.env.API_BASE_URL}/api/secure/dailystats?fromDate=${someGeneratedData}`
```

Not a formal review but this jumped out to me! Overall this looks exciting thanks for your work